expat/xmlwf/xmlwf.c

```diff
@@ -322,6 +322,14 @@ freeNotations(XmlwfUserData *data) {
   data->notationListHead = NULL;
 }
 
+static void
+cleanupUserData(XmlwfUserData *userData) {
+  if (userData->currentDoctypeName != NULL) {
+    free((void *) userData->currentDoctypeName);
```

Comment:
Not too important, but the cast to `void *`can probably be dropped. Especially if this person here https://stackoverflow.com/a/49040431 is correct, because we do include `stdlib.h`in `xmlwf.c`.

Reply:
PS: Looks like the Expat code base uses both with and without cast as of today:
```console
# git grep -F 'free(' | fgrep 'void *' | wc -l
12
# git grep -F 'free(' | fgrep -v 'void *' | wc -l
36
```
I guess that could use consistification some time.